src/app/Main/Body/QueryPage/QueryPageContent-Layout.component.tsx

```diff
@@ -15,22 +15,15 @@ import { QueryPageContentBarChartResultsComponent } from "./Results/Charts/Query
 const contentStyle = css`
     width: 0;
     height: inherit;

```

Comment:
i think this can be removed: from the inherit mdn docs: 'For non-inherited properties, this specifies a behavior that typically makes relatively little sense and you may consider using initial instead, or unset on the all property.'

Reply:
Done.